lib/identifier/uuid/uuid.rb

```diff
@@ -1,5 +1,17 @@
 module Identifier
   module UUID
+    MATCHER = %r{
```

Comment:
This appears to be a pattern, more than a `matcher`. Also, it's only a constant if it NEVER, EVER changes (like pi, or Planck's Constant).